### PR TITLE
Android remove curl

### PR DESCRIPTION
**Pull request by CedricGuillemet**
removed dep on curl
removed openssl sub repo
use Java for async loading from url
tested with marble tower
build time from 24min to 5min (CI)
arm64 ABI goes down from 16Mb to 11Mb (32% less !!)

### CODE REVIEW

File: Apps/Playground/Android/app/src/main/cpp/BabylonNativeNetwork.cpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    {
+        std::vector<uint8_t> content;
+        if (AndroidLoadAsset(url, content))
+        {
+            contentHandler(content);
+        }
+        else
+        {
+            DispatchLooperFunction<DispatchPayloadDirect>(g_mainThreadLooper, [url, contentHandler = std::move(contentHandler)](){
+                JNIEnv * env = AcquireEnvFromJVM(g_jvm);
```
Comment (ryantrem) [inline Apps/Playground/Android/app/src/main/cpp/BabylonNativeNetwork.cpp]: Which thread would this be getting called from? Is it the JS thread? If so, this part of the change will overlap with a more generalized solution that I believe @bghgary was going to implement for getting the JVM attached to the JS thread as needed (we need to do this for the ARCore integration).

Comment (bghgary) [reply Apps/Playground/Android/app/src/main/cpp/BabylonNativeNetwork.cpp]: That's right. @CedricGuillemet put a comment on DipatchLooperFunction letting me know the code to put in arcana.

Comment (ryantrem) [reply Apps/Playground/Android/app/src/main/cpp/BabylonNativeNetwork.cpp]: I wasn't talking about the looper scheduler, I was talking about attaching the JVM to the JS thread.

**Comment by bghgary** [comment]
CI not running. Closing to retry.

File: Apps/Playground/Android/app/src/main/cpp/BabylonNativeNetwork.cpp (hunk 1 above)
Comment (bghgary) [reply Apps/Playground/Android/app/src/main/cpp/BabylonNativeNetwork.cpp]: Oh, right. Yeah, we should use the new code that in the arcore branch.

**Comment by CedricGuillemet** [comment]
Replaced by @bghgary 's work on xmlhttprequest. Closing.

**Pull request closed without merging**